libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.applicationinsights;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.bot.applicationinsights.core.TelemetryInitializerMiddleware;
+import com.microsoft.bot.builder.BotTelemetryClient;
+import com.microsoft.bot.builder.TelemetryLoggerMiddleware;
+import com.microsoft.bot.builder.adapters.TestAdapter;
+import com.microsoft.bot.builder.adapters.TestFlow;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.ActivityTypes;
+
+public class TelemetryInitializerTests {
+
+    @Test
+    public void telemetryInitializerMiddlewareLogActivitiesEnabled() {
+    	TelemetryClient mockTelemetryClient = Mockito.mock(BotTelemetryClient.class);
+
+        TelemetryLoggerMiddleware telemetryLoggerMiddleware = new TelemetryLoggerMiddleware(mockTelemetryClient, false);
+
+        TestAdapter testAdapter = new TestAdapter()
+        		.use(new TelemetryInitializerMiddleware(telemetryLoggerMiddleware, true));
+
+        // Act
+        // Default case logging Send/Receive Activities
+        TestFlow testFlow = new TestFlow(testAdapter, turnContext -> {
```

Comment:
Don't set the TestFlow to a variable, check another unit test in Java and leave it as `new TestFlow(....`

Check [this](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/QnAMakerTests.java#L121) as example